server/routes/activitiesMigration/activitiesMigrationController.ts

```diff
@@ -124,6 +133,12 @@ export default class ActivitiesMigrationController {
     res.redirect('/activities-migration')
   }
 
+  async postActivatePrison(req: Request, res: Response): Promise<void> {
+    const { serviceName, prisonId } = req.query as { serviceName: string; prisonId: string }
+    await this.nomisPrisonerService.createServiceAgencySwitch(prisonId, serviceName, context(res))
+    res.redirect('/activities-migration/amend')
+  }
+
```

Comment:
If the prison is not turned on we show a button that calls NOMIS API to switch on the prison.